src/main/java/picard/fingerprint/CheckFingerprint.java

```diff
@@ -89,10 +93,10 @@
             "any number of genotypes; CheckFingerprint will use only those that are usable for fingerprinting.")
     public File GENOTYPES;
 
-    @Option(optional=true, doc = "If using VCF format genotypes, this parameter can be used to specify which sample's " +
-            "genotypes to use from the VCF file.  If not supplied the sample name from the BAM read group header " +
-            "is used instead.")
-    public String SAMPLE_ALIAS;
+    @Option(optional=true, doc = "This parameter can be used to specify which sample's genotypes to use from the " +
```

Comment:
you are breaking backwards compatibility, could you make SAMPLE_ALIAS the short name for the argument?